### PR TITLE
[MIG][10.0] #4943 website_blog_group_old_archives module

### PR DESCRIPTION
**Pull request by gfcapalbo**
Adds an option to group old blog posts on the right colum in blog.
A little different than the normal grouping option in odoo10 because it groups by month for the current month and then groups by year all the blogs of previous years.

depends on website_blog_layout_options , to be considered after merge of that module.
Only after and if  https://github.com/OCA/website/pull/504  is merged.

### CODE REVIEW

**Comment by rafaelbn** [comment]
WIP

- [ ] https://github.com/OCA/website/pull/504

File: website_blog_group_old_archives/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
```
Comment (nikul-serpentcs) [inline website_blog_group_old_archives/__init__.py:2]: @gfcapalbo Remove Copyright from all `__init__` file

File: website_blog_group_old_archives/views/templates.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
```
Comment (nikul-serpentcs) [inline website_blog_group_old_archives/views/templates.xml:2]: `odoo`

File: website_blog_group_old_archives/views/templates.xml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <data>
```
Comment (nikul-serpentcs) [inline website_blog_group_old_archives/views/templates.xml:3]: IMO remove tag

File: website_blog_group_old_archives/views/templates.xml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        <template id="opt_blog_rc_history_group_old_archive"                                                        
+            inherit_id="website_blog.opt_blog_rc_history">                                  
+            <xpath expr="//section[@class='mt32']" position="replace">
```
Comment (nikul-serpentcs) [inline website_blog_group_old_archives/views/templates.xml:6]: here use `hasclass`

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Pull request closed without merging**